build.sh

```diff
@@ -2,17 +2,5 @@
 
 DOCKER_IMAGE=mjdymalla/iso3166-library
 
-if [ $1 = "rebuild" ]
-then
-    docker build --no-cache -t $DOCKER_IMAGE .
-else
-    docker build -t $DOCKER_IMAGE .
-fi
-
-sudo apt install composer
-
-composer update
```

Comment:
You don't need to update, see comment regarding `COPY . /app` / `RUN composer install`